src/components/editor/ImportFileZone.jsx

```diff
@@ -7,7 +7,7 @@ import Ajv from 'ajv' // JSON schema validation
 const util = require('util') // for JSON schema validation errors
 import { Link } from 'react-router-dom'
 
-class StartingPoints extends Component {
+class ImportFileZone extends Component {
```

Comment:
I think surely they would be separate components used by ImportResourceTemplate (see my comment w/ design screenshot below). I named it with Zone precisely because it is associated with the DropZone component.